kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -35,55 +36,51 @@ def __init__(self, dataset_ingest_config_path):
                                               'output')
 
     def run(self, target_api_config_path, auto_transform=False,
-            use_async=False, target_url=DEFAULT_TARGET_URL):
+            use_async=False, target_url=DEFAULT_TARGET_URL,
+            log_level_name=None):
         """
         Entry point for data ingestion. Run ingestion in the top level
         exception handler so that exceptions are logged.
 
         See _run method for param description
         """
-        # Create logger
-        self._get_log_params(self.data_ingest_config)
+        # Get args, kwargs
+        frame = inspect.currentframe()
+        args, _, _, values = inspect.getargvalues(frame)
+        kwargs = {arg: values[arg] for arg in args[2:]}
```

Comment:
Huh? This is code that was already there... also it is used:

here to log ingest params - https://github.com/kids-first/kf-lib-data-ingest/blob/471ad052c3ef21edde8c5667fd11582d67067894/kf_lib_data_ingest/etl/ingest_pipeline.py#L70

here as input to _run-  https://github.com/kids-first/kf-lib-data-ingest/blob/471ad052c3ef21edde8c5667fd11582d67067894/kf_lib_data_ingest/etl/ingest_pipeline.py#L76      

For the latter, as I've added new kwargs to the CLI over time, I'm getting tired of forgetting to also add them to the call to _run. So now i'm just passing them in as a dict instead of individually listing out all the kwargs

Reply:
Huh. I searched for it and didn't see anything a moment ago. I blame github.